cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -329,6 +329,7 @@ public String parseLogging(Logging logging, ConfigMap externalCm) {
 
         } else {
             // field is not in the cluster CM
+            this.setLogging(new InlineLogging());
```

Comment:
this isn't the right place to do this, I think. If the default is an `InlineLogging` that should be set in `fromCrd()`. After `fromCrd()` is call we should treat the `KafkaCluster` (or whatever subclass) as immutable.